src/hooks/base.js

```diff
@@ -1,7 +1,7 @@
 import EventEmitter from 'eventemitter3';
 
 export default class extends EventEmitter {
-    on(eventName, elementSelector, handler) {
+    subscribe(eventName, elementSelector, handler) {
```

Comment:
Why are we making this change ? Is there some kind of name collusion?

Reply:
Yep, that's why some tests were failing on cornerstone. EventEmitter has method with name _on_